st2common/st2common/util/virtualenvs.py

```diff
@@ -166,8 +166,12 @@ def install_requirements(virtualenv_path, requirements_file_path):
     exit_code, stdout, stderr = run_command(cmd=cmd, env=env)
 
     if exit_code != 0:
-        raise Exception('Failed to install requirements from "%s": %s (stderr: %s)' %
-                        (requirements_file_path, stdout, stderr))
+        def safetystring(s):
```

Comment:
I would be happy to but I don't have any idea how to get started with that.  I don't see any existing tests of the install_requirements routine, or of anything referencing virtualenv.  Nor do I really know how to do what needs to be done to test this.  :-(